### PR TITLE
Feature/custom redirect domain

### PR DESCRIPTION
**Pull request by slocka**

### CODE REVIEW

File: extensions/amp-skimlinks/0.1/skim-options.js [hunk 1]
```diff
@@ -147,6 +150,29 @@ function getInternalDomains_(docInfo) {
...
 }
 
+/**
+ *
+ * @param {!Element} element
+ * @return {string}
+ */
+function getWaypointBaseUrl(element) {
+  let customSubDomain = element.getAttribute('custom-redirect-domain');
+  if (customSubDomain) {
```
Comment (pedroabreu) [inline extensions/amp-skimlinks/0.1/skim-options.js:160]: Is it worth checking if it's a valid domain format ?

**Review by pedroabreu — APPROVED** [review]
Looks good 👍

File: extensions/amp-skimlinks/0.1/skim-options.js [hunk 2]
```diff
@@ -147,6 +150,29 @@ function getInternalDomains_(docInfo) {
...
+    // Remove potential HTTP protocol
+    customSubDomain = customSubDomain.replace(/^\/\/|^https?:\/\//, '');
+    // Remove potential trailing slash
+    customSubDomain = endsWith(customSubDomain, '/') ?
```
Comment (lepunk) [inline extensions/amp-skimlinks/0.1/skim-options.js]: this doesn't seem to handle the case of custom.subdomain.com/beer
not sure if its a realistic concern tho

**Review by lepunk — APPROVED** [review]
looks all right besides the minor concern

File: extensions/amp-skimlinks/0.1/skim-options.js (hunk 2 above)
Comment (slocka) [reply extensions/amp-skimlinks/0.1/skim-options.js]: `custom.subdomain.com/beer` would work, this line is just removing the trailing slash if there is one.

**Pull request closed without merging**